port/port_posix_sse.cc

```diff
@@ -54,7 +54,7 @@ static inline bool HaveSSE42() {
   __cpuid(cpu_info, 1);
   return (cpu_info[2] & (1 << 20)) != 0;
 #elif defined(__GNUC__)
-  unsigned int eax, ebx, ecx, edx;
+  unsigned int eax, ebx, ecx = 0, edx;
   __get_cpuid(1, &eax, &ebx, &ecx, &edx);
   return (ecx & (1 << 20)) != 0;
```

Comment:
ah bingo. @TheBlueMatt  see how the sse4 stuff in bitcoin master uses __get_cpuid. please do that.